### DISCUSSION TITLE
Retrieving std::vector<Real> material property as an auxvariable.

### DISCUSSION THREAD

**Discussion by steam-simulator** [idaholab/moose#25747] [Q&A Modules: General]
Hello, 
Is there a way to retrieve a declared `std::vector<Real>` material property as an auxvariable? Currently, it is possible for only `Real` material properties. Thx.

**Comment by GiudGiud** [comment]
You can do it component by component using the MaterialStdVectorAux I think

**Reply by GiudGiud** [reply]
See this documentation
https://mooseframework.inl.gov/source/auxkernels/MaterialStdVectorAux.html
